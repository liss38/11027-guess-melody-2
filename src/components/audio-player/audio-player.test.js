import React from 'react';
import renderer from 'react-test-renderer';
import AudioPlayer from './audio-player.jsx';

it(`AudioPlayer is rendered`, () => {
  const tree = renderer
    .create(<AudioPlayer
      src={``}
      isPlaying={false}
      onPlayButtonClick={jest.fn()}
    />,
    {
      createNodeMock: (element) => {
        if (element.type === `audio`) {
          return {};
        }

        return null;
      }
    })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
