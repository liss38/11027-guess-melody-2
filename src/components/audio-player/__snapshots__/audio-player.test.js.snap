// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AudioPlayer is rendered 1`] = `
Array [
  <button
    className="track__button track__button--play js-track-button"
    disabled={true}
    onClick={[Function]}
    type="button"
  />,
  <div
    className="track__status"
  >
    <audio />
  </div>,
]
`;
